components/base/kinematicbase/ptgKinematics.go

```diff
@@ -169,18 +182,24 @@ func (ptgk *ptgBaseKinematics) Kinematics() referenceframe.Frame {
 func (ptgk *ptgBaseKinematics) CurrentInputs(ctx context.Context) ([]referenceframe.Input, error) {
 	ptgk.inputLock.RLock()
 	defer ptgk.inputLock.RUnlock()
-	return ptgk.currentState.currentInputs, nil
+
+	planningFrameInputs := ptgk.currentState.currentInputs
+	return planningFrameInputs, nil
 }
 
 func (ptgk *ptgBaseKinematics) ExecutionState(ctx context.Context) (motionplan.ExecutionState, error) {
 	if ptgk.Localizer == nil {
 		return motionplan.ExecutionState{}, errors.New("cannot call ExecutionState on a base without a localizer")
 	}
 
-	actualPIF, err := ptgk.CurrentPosition(ctx)
+	actualPIF, err := ptgk.Localizer.CurrentPosition(ctx)
 	if err != nil {
 		return motionplan.ExecutionState{}, err
 	}
+	actualPIFInRadians := referenceframe.NewPoseInFrame(
```

Comment:
`actualPIF` originally has its orientation's theta be in degrees. We want it to be in radians.
We call on the executionState in `getTransientDetections` so that we may inform the values of the `inputMap`.
To create the `inputMap`'s values for the kinematicBase we turn the the pose into a list of inputs.

We do the converse in `CheckPlan`, i.e. we turn a list of inputs into a pose. The inputs we are working with within `CheckPlan` are describe a pose's orientation in radians. This ensures that orientation is handled properly.

Reply:
This is just a Pose so this should be a quaternion on the backend both before and after this code, no?

Reply:
I've convinced myself this is un-necessary.
My thinking which led me down the path of thinking this _was_ necessary was based on the fact that the poseFrame's transform method expects the incoming inputs to represent a pose whose orientation's theta is already in radians.